1.8/administration/overlay-networks/index.md

```diff
@@ -4,7 +4,8 @@ post_title: Overlay Networks
 menu_order: 11.5
 ---
 
-DC/OS enables virtual networking through the use of overlay networks. DC/OS overlay networks enable you to provide each container in the system with a unique IP address (“IP-per-container”) with isolation guarantees amongst subnets. DC/OS overlay networks offer the following advantages:
+DC/OS supports virtual networking through CNI (Container Network Interface) for `MesosContainerizer`, and Docker libnetwork for `DockerContainerizer`. The DC/OS overlay network provides an out-of-the-box
```

Comment:
How about:

"The DC/OS overlay network feature is an out-of-the-box virtual networking solution that provides an ip-per-container for Mesos and Docker containers alike. The DC/OS overlay network uses CNI (Container Network Interface) for the `MesosContainerizer` and Docker libnetwork for the `DockerContainerizer`. 

DC/OS overlay networks allow containers launched through the `MesosContainerizer` or `DockerContainerizer` to co-exist on the same IP network, allocating each container their own unique IP address. DC/OS overlay networks offer the following advantages:"